docs/source/changes/27.documentation.yaml

```diff
@@ -0,0 +1,8 @@
+category: added
+summary: "Basic documentation"
+description: |
+  LAPIS now includes a basic documentation about the different components and
+  concepts, importing jobs and pools, the logging process and database structure
+  as well as the command line interface.
+pull requests:
+  - 27
```

Comment:
Is there a reason why this doesn't have a version number yet? Or was it merged much later than the others?

Reply:
So you do not remember that you recently reviewed the huge bunch of changes 😄